src/main/java/seedu/address/model/step/Step.java

```diff
@@ -0,0 +1,4 @@
+package seedu.address.model.step;
+
+public class step {
```

Comment:
class name should start with capital letter

Reply:
wait ah i forget to commit. I finished alr

Reply:
how come my travis check failed?

Reply:
i forget to commit my code. Fixed alr Thanks